js/extensions/cadastrapp/components/Main.jsx

```diff
@@ -0,0 +1,386 @@
+
+            alert("The user selected a plot by a linestring tool. Adding (2) random to plot data to 'Plots Selection' section");
+            selectionData[activeSelectionTab].push(randomPlot());
+            selectionData[activeSelectionTab].push(randomPlot());
+            setPlotSelectionData(selectionData);
+            break;
+
+        case "select-by-polygon":
+            if (selectionData.length === 0) {selectionData = [[]];}
+
+            alert("The user selected a plot by a polygon tool. Adding (3) random to plot data to 'Plots Selection' section");
+            selectionData[activeSelectionTab].push(randomPlot());
+            selectionData[activeSelectionTab].push(randomPlot());
+            selectionData[activeSelectionTab].push(randomPlot());
+            setPlotSelectionData(selectionData);
+            break;
+
+        case "unit-de-fonc":
+            alert("The user clicks on a selection zoom in the map and he is redirected an external web page in a new browser tab");
+            url = "https://portail.sig.rennesmetropole.fr/mapfishapp/ws/addons/cadastrapp/html/ficheUniteFonciere.html";
+            window.open(url, '_blank');
+            break;
+
+        case "search-plots":
+            setActiveToolbar("search-plots");
+            setIsPlotsSearchShown(true);
+            setIsOwnersSearchShown(false);
+            setIsCoownershipSearchShown(false);
+            setIsWelcomeShown(false);
+            break;
+
+        case "search-owners":
+            setActiveToolbar("search-owners");
+            setIsOwnersSearchShown(true);
+            setIsPlotsSearchShown(false);
+            setIsCoownershipSearchShown(false);
+            setIsWelcomeShown(false);
+            break;
+
+        case "coownership":
+            setActiveToolbar("coownership");
+            setIsCoownershipSearchShown(true);
+            setIsPlotsSearchShown(false);
+            setIsOwnersSearchShown(false);
+            setIsWelcomeShown(false);
+            break;
+
+        case "request-form":
+            setIsRequestFormShown(true);
+            break;
+
+        case "preferences":
+            setIsPreferencesModalShown(true);
+            break;
+
+        case "help":
+            url = "https://portail.sig.rennesmetropole.fr/actus/aide/cadastre";
+            window.open(url, '_blank');
+            break;
+        default:
+            break;
+        }
+    };
+
+    const handlePlotsSelectionClick = (action) => {
+        switch (action) {
+        case "information-form":
+            setIsInformationFormShown(true);
+            break;
+        case "export":
+            alert("Selection export is clicked");
+            break;
+        case "zoom":
+            alert("Selection zoom is clicked");
+            break;
+        case "landry":
+            alert("You clicked on landry information button, you will redirecting an external service in a new tab");
```

Comment:
Can you take https://github.com/georchestra/mapstore2-cadastrapp/issues/12#issuecomment-704301264 into account ? thanks !

Reply:
@landryb don't worry this is just a first PR. We will address this comment when we will work on issue #12, as soon as we are informed about the new string to use.

Reply:
well, i think we've settled on `unite_fonciere`, no ?